### PR TITLE
[8.0] fix (MonitoringDB): make a MonitoringDB singleton

### PR DESCRIPTION
**Pull request by chaen**
Take this very simple script that does basically nothing

```python
from DIRAC.Core.Base.Script import parseCommandLine
parseCommandLine()
from DIRAC.DataManagementSystem.Client.FTS3Operation import FTS3Operation

for i in range(100):
    FTS3Operation()
```

It takes 45 seconds to execute on a server.
The reason is that each of these `FTS3Operation` will end up initializing a`MonitoringDB` instance, which will connect to `ElasticSeach`, list the indexes, and what not.... 

Unacceptable :unamused: 

So I just make a `MonitoringDB` singleton. With that, the script takes 2 seconds
I have full trust that the original author thought about thread safety.... (looks ok, from a quick glance)

BEGINRELEASENOTES
*Monitoring
CHANGE: introduce gMonitoringDB global instance

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/MonitoringSystem/Client/ServerUtils.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-    serverName = "Monitoring/Monitoring"
-    MonitoringDB = None
     try:
-        from DIRAC.MonitoringSystem.DB.MonitoringDB import MonitoringDB
+        if gMonitoringDB and gMonitoringDB._connected:
+            return gMonitoringDB
     except Exception:
         pass
-    return getDBOrClient(MonitoringDB, serverName)
+    return Client(url="Monitoring/Monitoring")
```
Comment (fstagni) [inline src/DIRAC/MonitoringSystem/Client/ServerUtils.py:16]: why this change?

Comment (chaen) [reply src/DIRAC/MonitoringSystem/Client/ServerUtils.py:16]: Sorry I am not sure I understand the question ?

**Review by fstagni — APPROVED** [review]

File: src/DIRAC/MonitoringSystem/Client/ServerUtils.py (hunk 1 above)
Comment (fstagni) [reply src/DIRAC/MonitoringSystem/Client/ServerUtils.py:16]: OK I see it.

**Comment by fstagni** [comment]
Still a draft?
